### ISSUE TITLE
FE | Profile | Accessibility |  Use TTY Prop in Telephone Component

### ISSUE THREAD

**Issue by SKasimow** [department-of-veterans-affairs/va.gov-team#46195]
## Background

TTY should have an aria label to better support screenreader users.
Also as part of this effort, pls update any TTY phone numbers that **DON'T** show "711".
2 examples can be found in #49843)

## Tasks
- [ ] Make Changes based on designs
- [ ] Conduct Unit tests 
- [ ] Run E2E tests covering current code and regression
- [ ] Notify PM if this needs to be deployed to Staging behind feature flag
  

## Acceptance Criteria
- [ ] Complete all unit testing
- [ ] End-to-end tests showing 0 violations
- [ ] All axe checks pass
- [ ] Notify Product Manager that the change is ready to be validated on Staging
- [ ] Incorporate any changes resulting from Staging review & re-test & have re-validated
- [ ] Confirm PM has opened a new ticket to deploy changes to Production using feature flag, if needed
- [ ] Scrubbing the profile codebase for all TTY usages,
- [ ] Verifying that they are using the correct va-telephone component with tty prop
- [ ] Updating out of date instances of TTY

## Validation
- Who can validate this ticket? (FE, BE, Design, PM)? PM or Design
- How can this work be validated? Staging
- What updates need to be made (e.g. product outline, use cases, contact center guide) - n/a

## Associated Tickets

Note: existing telephone component **does** have a specific TTY property

**Comment by andaleliz** [comment]
@SKasimow this is probably already on your to do list, but just in case it's not: it looks like we need some more details on this ticket based on the outcome of [Angela's audit finding](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/identity-personalization/profile/accessibility/accessibility-audit-updates.md#tty-direct-deposit), as well as tasks and AC.

**Comment by SKasimow** [comment]
@andaleliz - yup I talked with Adam about this earlier and needs some further discussion/clarification for the additional details that need to be added. I'll note it as "placeholder" for now.

**Comment by SKasimow** [comment]
@adamwhitlock1 - can you mirror how TTY aria labels are set up in other sections such as on the Personal info page "How to fix an error..." drop-down?

**Comment by adamwhitlock1** [comment]
This is how the personal information additional info section shows the TTY. It is built as a hard coded anchor element instead of a `va-telephone` design system component.

![Screen Shot 2022-09-28 at 11 51 18 AM](https://user-images.githubusercontent.com/8332986/192854129-3ce1e36a-8e64-47a7-b3ad-d569b189d0c5.png)

**Comment by SKasimow** [comment]
@jcatergovy noted that you will also be working on #49843 as part of this ticket. LMK if we should bump on points.

**Comment by jcatergovy** [comment]
@cgednalske @SKasimow ready for validation in staging

**Comment by SKasimow** [comment]
Hey @jcatergovy - I was able to verify that the 711 number shows correctly on the 2 examples I provided in the linked ticket but not sure whether we need someone from accessibility to test for the aria label or whether it is sufficient to just show that it has been added to code.

@adamwhitlock1 wdyt?

**Comment by jcatergovy** [comment]
@SKasimow I was able to verify it's there using the inspector in the browser.  Is that what you meant by show that it's in code?
